examples/storm-mqtt-examples/src/main/java/org/apache/storm/mqtt/examples/CustomMessageMapper.java

```diff
@@ -15,35 +15,77 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+
 package org.apache.storm.mqtt.examples;
 
-import org.apache.storm.tuple.Fields;
-import org.apache.storm.tuple.Values;
 import org.apache.storm.mqtt.MqttMessage;
 import org.apache.storm.mqtt.MqttMessageMapper;
+import org.apache.storm.tuple.Fields;
+import org.apache.storm.tuple.Values;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 /**
  * Given a topic name: "users/{user}/{location}/{deviceId}"
  * and a payload of "{temperature}/{humidity}"
- * emits a tuple containing user(String), deviceId(String), location(String), temperature(float), humidity(float)
- *
+ * emits a tuple containing
+ * {@code user(String), deviceId(String), location(String), temperature(float),
+ * humidity(float)}.
  */
-public class CustomMessageMapper implements MqttMessageMapper {
-    private static final Logger LOG = LoggerFactory.getLogger(CustomMessageMapper.class);
-
+public final class CustomMessageMapper implements MqttMessageMapper {
+    private static final long serialVersionUID = 1L;
+    /**
```

Comment:
I guess checkstyle doesn't require fields to be commented, and the name of fields are self-descriptive. Better to revert them.

Reply:
Correct. I assumed that `checkstyle:check` produces the same output as the checkstyle configuration run during `mvn validate`, but the former is much stricter.